src/application/res/backend/os_dashboard_api.py

```diff
@@ -87,6 +87,7 @@ def get_iframes(self):
                 iframe_data.append({
                     'title': title,
                     #'type': vis_type,
```

Comment:
```suggestion
```

At @oninach this was our code. I think, we just forgot to remove this line, when we replaced visualizations by dashboards. Let's do it know.